ui/pages/settings/contact-list-tab/add-contact/add-contact.component.js

```diff
@@ -24,28 +25,32 @@ export default class AddContact extends PureComponent {
     qrCodeData:
       PropTypes.object /* eslint-disable-line react/no-unused-prop-types */,
     qrCodeDetected: PropTypes.func,
+    ensResolution: PropTypes.string,
+    ensError: PropTypes.string,
+    resetResolution: PropTypes.func,
   };
 
   state = {
     newName: '',
     ethAddress: '',
-    ensAddress: '',
     error: '',
-    ensError: '',
+    input: '',
   };
 
   constructor(props) {
     super(props);
-    this.dValidate = debounce(this.validate, 1000);
+    this.dValidate = debounce(this.validate, 500);
```

Comment:
Just curious - why was this changed? I'm guessing this felt more responsive when testing?

Reply:
Yes. the full second delay seemed to be larger than necessary.